processing/data_collection/gazette/spiders/rj_campos_goytacazes.py

```diff
@@ -0,0 +1,45 @@
+import re
+import dateparser
+
+from datetime import datetime
+from scrapy import Request
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+class RjCampoGoytacazesSpider(BaseGazetteSpider):
+    GAZETTE_ELEMENT_CSS = 'ul.ul-licitacoes li'
+    MUNICIPALITY_ID = '3301009'
+
+    allowed_domains = ['www.campos.rj.gov.br']
+    name = 'rj_campos_goytacazes'
+    start_urls = ['https://www.campos.rj.gov.br/diario-oficial.php?PGpagina=1&PGporPagina=15']
+
+    def parse(self, response):
+        """
+        @url https://www.campos.rj.gov.br/diario-oficial.php?PGpagina=1&PGporPagina=15
+        @returns requests 1
```

Comment:
```@returns items 15 15```

Reply:
Addressed at https://github.com/okfn-brasil/querido-diario/pull/235/commits/7c34cceb0512098b8071618937344b4d14bbfd59.

Reply:
@chamun you need to update this branch with the commit.